examples/inngest/package.json

```diff
@@ -19,6 +19,7 @@
     "@types/react-dom": "18.2.7",
     "concurrently": "^8.2.1",
     "encoding": "^0.1.13",
-    "typescript": "5.2.2"
+    "inngest-cli": "latest",
```

Comment:
@djfarrelly Is it `inngest` or `inngest-cli`?
x-ref: https://www.npmjs.com/package/inngest

Reply:
Yep! For additional clarification, the inngest package is the SDK and the inngest-cli is used to spin up the dev server which communicates with the Next.js application.

Reply:
I updated the `inngest` package above to `3.x` - I believe the CLI is correct at latest.